configure.ac

```diff
@@ -202,8 +202,10 @@ AS_IF([ test ${WIN32} -eq 1 ],[
     AS_IF([ test "x${CC64}" = "x" ], [
       AS_IF([ test "$ARCH" = "i386" ],
         [T_CC64=x86_64-w64-mingw32-gcc
+        COMP_ARCH='x64'
```

Comment:
It's a confusing one - Microsoft call it x64 (cf. the other side of the `AS_IF`!). However, I think you're right and I've changed it - if we're building 32-bit mingw, then we call it x86_64 in the display (as you say, because Cygwin and GCC do!); if we're building 32-bit MSVC, then we call it x64 (because Microsoft does) and then the complaint always matches the name of the missing compiler! 🤯